src/thor/dijkstras.cc

```diff
@@ -157,10 +157,11 @@ void Dijkstras::ExpandForward(GraphReader& graphreader,
       }
     }
 
-    // Compute the cost to the end of this edge
+    // Compute the cost and path distance to the end of this edge
     Cost transition_cost = costing_->TransitionCost(directededge, nodeinfo, pred);
     Cost newcost = pred.cost() + costing_->EdgeCost(directededge, tile, offset_time.second_of_week) +
                    transition_cost;
+    uint32_t path_dist = pred.path_distance() + directededge->length();
```

Comment:
I chose `path_distance` instead of `distance`, I figured it's semantically more correct. I know we discussed adding this directly to the isochrone's `UpdateIsoTile()` and leaving the dijkstra alone.. But the edge labels have no way to set `path_distance` after they're created and supporting that would need quite a lot more change than simply tracking the path distance in the dijkstra.

Reply:
I did need to add an overload on the `BDEdgeLabel`s constructor to support this, see below..

Reply:
no this was the right fix. actually all of the algorithms should be tracking this. really what we should do is remove it from the  label and add it to the costing and let it be tracked in there.